addons/ofxEmscripten/src/ofxAppEmscriptenWindow.cpp

```diff
@@ -89,14 +101,27 @@ void ofxAppEmscriptenWindow::setup(const ofGLESWindowSettings & settings){
 		ofLogError() << "couldn't get configs";
 		return;
 	}
+    
+    // Choose the config based on our attribute list
+    // Try higher EGL_DEPTH_SIZE first
+    for(int i = 0; i < depthPreference.size(); i++){
+        // Set EGL_DEPTH_SIZE
+        attribList[attribListDepthIndex] = depthPreference[i];
+        
+        // Try out that depth value
+        if ( !eglChooseConfig(display, &attribList[0], &config, 1, &numConfigs) ){
+
+            // Finally fail like we did before if no preference works 
+            if( depthPreference[i] == EGL_DONT_CARE ){
+                ofLogError() << "couldn't choose display";
+                return;
+            }
 
-	ofLogNotice("ofxAppEmscriptenWindow") << "Got " << numConfigs << " display configs";
-
-	// Choose config
-	if ( !eglChooseConfig(display, attribList, &config, 1, &numConfigs) ){
-		ofLogError() << "couldn't choose display";
-		return;
-	}
+        }else{
+            ofLogNotice() << " depth is successfully set to " << depthPreference[i] << endl;
```

Comment:
Can you remove this notice or at least set it to verbose?

Reply:
Sure. Just pushed!